### ISSUE TITLE
Minor bug in documentation

### ISSUE THREAD

**Issue by arvindmohan** [google/jax#2015]
In the autodiff cookbook,

"To implement hessian, we could have used jacrev(jacrev(f)) or jacrev(jacfwd(f)) or any other composition of the two. But forward-over-reverse is typically the most efficient. That's because in the inner Jacobian computation we're often differentiating a function wide Jacobian (maybe like a loss function  𝑓:ℝ𝑛→ℝ ),"

I believe this should be "jacfwd(jacrev(f)) or jacrev(jacfwd(f))".

P.S. Thanks for the excellent work. Hessians are a breeze !

**Comment by ziyadedher** [comment]
I think what the documentation is trying to say is that you can compute the Hessian using two Jacobian computation operations `jac*(jac*(f))`, which is true. So what you're saying would work, but the current documentation is correct as well!

Something to note is that `jacfwd` and `jacrev` always compute the same value, even though they are different under-the-hood. [Check this out for more details](https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#Jacobians-and-Hessians-using-jacfwd-and-jacrev).

Also, you can use always use [`jax.hessian`](https://jax.readthedocs.io/en/latest/jax.html#jax.hessian) to compute Hessians as well; however, [the current implementation](https://jax.readthedocs.io/en/latest/_modules/jax/api.html#hessian) uses `jacfwd(jacrev( . ))` which might be less efficient than choosing a composition depending on your use-case.

**Comment by arvindmohan** [comment]
I see! Thanks for the clarification - I figured it was a typo because the fwd/rev nomenclature used was not consistent with the previous sentence. It makes sense now.
